### PR TITLE
completed Two Pointer 2 -- Please review

### PR DESCRIPTION
**Pull request by VenkateswaraRajuDatla**

### CODE REVIEW

File: Sample.java [hunk 1]
```diff
@@ -1,7 +1,86 @@
...
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Three line explanation of solution in plain english
+//merge sorted array
+//Time o(n) n is the length of the array nums1
+//space o(1)
+class Solution {
+    public void merge(int[] nums1, int m, int[] nums2, int n) {
+        if(n == 0)
```
Comment (iivaishnavii) [inline Sample.java:7]: Good job handling the edge cases

File: Sample.java [hunk 2]
```diff
@@ -1,7 +1,86 @@
...
+                slowPtr++;
+                nums[slowPtr] = nums[fastPtr];
+                fastPtr++;
+            }
+        }
+      return slowPtr+1;  
+    }
+}
+
+//Time o(m+n)
```
Comment (iivaishnavii) [inline Sample.java:63]: Binary search - will we go through all elements?

Comment (VenkateswaraRajuDatla) [reply Sample.java:63]: binary search - we will go through the half of the elements.

In this problem,
In the worst case, we will go through m + n elements in the i/p matrix.

**Comment by iivaishnavii** [comment]
Great job, Raju!!
